server/base/src/main/java/org/apache/accumulo/server/compaction/FileCompactor.java

```diff
@@ -279,7 +287,10 @@ public CompactionStats call() throws IOException, CompactionCanceledException {
       log.debug("Compaction canceled {}", extent);
       throw e;
     } catch (IOException | RuntimeException e) {
-      log.error("{}", e.getMessage(), e);
+      log.error("Compaction error. Compaction info: "
+          + "extent: {}, input files: {}, output file: {}, iterators: {}, start date: {}. Error message: {}",
+          getExtent(), getFileNamesToCompact(), getOutputFileName(), getIterators(),
+          threadStartDate, e.getMessage(), e);
```

Comment:
Is `e.getMessage()` along with `e` redundant.  If the exception is logged with the stack trace it will include the message as well.

Reply:
Yes, I agree. That is how it was originally logged, so I left those in. Wasn't sure if there was some weird reason for having both. I'll update in my next commit.

Reply:
It would be redundant, could drop that.